### PR TITLE
LPS-155309 | master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
**Background**
Create automated tests for the Frontend Dataset component.

**Test Plan**
Given Sample FDS portlet deployed
And entering on a page which uses FDS
When there are several filters already established
Then all of them can be removed at once
And the results will be displayed updated accordingly to it
And the Filters Summary boxes will be updated accordingly to it

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-155309

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase [hunk 1]
```diff
@@ -45,6 +45,30 @@ definition {
...
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+		}
+
+		task ("Then all of them can be removed at once") {
+			ClickNoError(locator1 = "FrontendDataSet#REMOVE_ALL_FILTERS");
+		}
+
+		task ("Then the results will be displayed updated accordingly to it") {
+			AssertElementPresent(locator1 = "FrontendDataSet#TABLE_BODY");
+		}
```
Comment (susanchen3) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase:63]: same as: https://github.com/liferay-frontend/liferay-portal/pull/2427#discussion_r907715654

**Review by susanchen3 — CHANGES_REQUESTED** [review]
Some minor changes :)

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
@susanchen3 Already fix the test.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path [hunk 2]
```diff
@@ -113,6 +118,11 @@
 	<td>//div[@class='dropdown-menu show']//*[@type="checkbox" and @aria-label='${key_status}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>RESET_FILTERS</td>
+	<td>//*[@class='tbar-item']//button[@class='btn btn-unstyled']</td>
+	<td></td>
```
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path:124]: Can we have the path be like `//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), 'Reset Filters')]` instead? :)

Comment (WYZ-Issac) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path:124]: OK, I will update the path and re-pull the commit.

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
@susanchen3 Fixed.

**Comment by susanchen3** [comment]
ci:forward

**Comment by susanchen3** [comment]
ci:forward

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**